common/namespace.yaml

```
@@ -18,4 +18,4 @@ namespaces:
     title: Table data types
   - doc: data types for different types of sparse matrices
     source: sparse.yaml
-  version: 1.1.3
+  version: 1.2.0
```

```suggestion
  version: 1.2.0a
```

The "master" branch should have a version that differentiates it from public releases. I will write up a release process guide in a separate PR.

---

Release notes should also be updated.